.gitignore

```diff
@@ -84,4 +84,6 @@ util/harvey.raw
 **/*.h.gch
 
 .vscode
-linux_amd64/
\ No newline at end of file
+linux_amd64/
```

Comment:
should this be / since the other one is too?

Reply:
changed. could you reapprove?